deploy/roles/neo4j/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 # main.yml
 ---
 - name: restart neo4j
-  command: "{{ neo4j_home_dir }}/bin/neo4j restart"
+  command: neo4j restart"
```

Comment:
@olivermel There's an extra trailing `"` at the end now, please remove that.  Can you re-run the script in vagrant by doing:
`vagrant destroy neo4j1`
`vagrant up neo4j1`
To ensure there are no syntax errors?